runtime/fixtures/runtime_test.dart

```diff
@@ -115,3 +114,30 @@ void testCanConvertListOfInts(List<int> args){
                 args[2] == 3 &&
                 args[3] == 4);
 }
+
+bool didCallRegistrantBeforeEntrypoint = false;
+
+// Test the Dart plugin registrant.
+// _PluginRegistrant requires the entrypoint annotation, so the compiler doesn't tree shake it.
```

Comment:
nit: we don't have this comment elsewhere for these annotations in this file.

Reply:
Done